src/globals.css

```diff
@@ -503,3 +503,7 @@ div[id^="headlessui-menu-items"] {
   @apply text-xl;
   @apply dark:border-neutral-700/80;
 }
+
+.gql-focus-visible {
```

Comment:
The design is kinda minimalist / brutalist and it doesn't specify the focus style, so I went with the simplest non-default one. Would've kept the default but blue clashes with our colors.

<img width="516" alt="image" src="https://github.com/user-attachments/assets/1209794f-8d13-4fe1-a74e-d69dd1be00ed" />